Xamarin.PropertyEditing.Mac/Controls/BaseNumericEditorControl.cs

```diff
@@ -91,10 +91,10 @@ protected override void SetEnabled ()
 		protected override void UpdateAccessibilityValues ()
 		{
 			NumericEditor.AccessibilityEnabled = NumericEditor.Enabled;
-			NumericEditor.AccessibilityTitle = ViewModel.Property.Name + " Numeric Editor"; // TODO Localization
+			NumericEditor.AccessibilityTitle = ViewModel.Property.Name + string.Format (" {0} {1}", LocalizationResources.AccessibilityNumeric, LocalizationResources.AccessibilityEditor);
```

Comment:
We should translate whole phrases, not individual words. "Numeric Editor" is the translatable phrase here!

Reply:
fixed